Swabbr.Core/Notifications/NotificationAction.cs

```diff
@@ -9,44 +9,39 @@
     /// </remarks>
     public enum NotificationAction
     {
-        /// <summary>
-        ///     Indicates that a followed profile is live.
-        /// </summary>
-        FollowedProfileLive = 0,
-
         /// <summary>
         ///     Indicates a followed profile posted a new vlog.
         /// </summary>
-        FollowedProfileVlogPosted = 1,
+        FollowedProfileVlogPosted = 0,
```

Comment:
This not causing any issues on the client side?

Reply:
No, the client currently maps from string. An integer was added in the previous PR (#179) which has not yet been used in the client side.